spython/logger/compatibility.py

```diff
@@ -1,6 +1,13 @@
+# Copyright (C) 2019-2019 Alexander Grund.
+
```

Comment:
I think you might have issue with the date here :)  If I make future additions, how does that work? it's weird to have a different header here.

Reply:
The date is fine. This way you can update the 2nd number only.   
I'm no expert in those Copyright lines but AFAIK you'd just add your name. But I'd probably just drop those lines. Going further I'd just remove the license headers too and just keep the `LICENSE` file:

> If it is not possible or desirable to put the notice in a particular file, then You may include the notice in a location (such as a LICENSE file in a relevant directory) where a recipient would be likely to look for such a notice.

Reply:
I'm not an expert either, which is what makes me uncomfortable. I had a colleague that accepted a weird license header and it led to not being able to change to a different derivative of the license because the code base wasn't entirely under his name. If you are ok with ok, I'd be more comfortable removing the header here, and I'd like to suggest adding a contributors file for your (and previous contributors) name. It might be unecessary since it's represented in GitHub commits, so I'll leave this to you!

Reply:
And apologies for asking you to add in the first place, I didn't think it fully through (and need to read more about best practices for this).

Reply:
I removed those first lines, so you can merge this I guess. Do the license header changes in a follow-up to keep this small(ish)